compiler/modules/front/stmt.bal

```diff
@@ -369,43 +359,53 @@ function addBindings(BindingChain? bindingLimit, BindingChain? bodyBindings, int
     return newBindings;
 }
 
+type ScopedStmt s:MatchStmt|s:WhileStmt|s:ForeachStmt|s:IfElseStmt;
```

Comment:
previous place is better since `codeGenScope` also use this.

Reply:
fixed